.eslintrc.js

```diff
@@ -7,5 +7,14 @@ module.exports = {
   "env": {
     "browser": true,
     "jquery": true
+  },
+  "rules": {
+    "no-use-before-define": "off",
+    "arrow-body-style": "off",
+    "arrow-parens": "off",
+    "semi": "off",
+    "max-len": "off",
+    "comma-dangle": "off",
+    "func-names": "off"
```

Comment:
Since this impacts the ease of debugging would you be willing to consider turning this one back on?